3.12/graphs-general-graphs.md

```diff
@@ -7,16 +7,60 @@ General Graphs
 ==============
 
 This chapter describes the [general-graph](graphs.html) module.
-It allows you to define a graph that is spread across several edge and document collections.
-This allows you to structure your models in line with your domain and group them logically in collections giving you the power to query them in the same graph queries.
-There is no need to include the referenced collections within the query, this module will handle it for you.
+It allows you to define a graph that is spread across several edge and document
+collections.
+This allows you to structure your models in line with your domain and group
+them logically in collections giving you the power to query them in the same
+graph queries.
+There is no need to include the referenced collections within the query, this
+module will handle it for you.
 
 New to ArangoDB? Take the free
 [ArangoDB Graph Course](https://www.arangodb.com/arangodb-graph-course){:target="_blank"}
 for freshers.
 
-Three Steps to create a graph
------------------------------
+## Getting started
+
+### Create a GeneralGraph using the Web Interface
+
+The Web Interface (also called Web UI) allows you to easily create and manage
+GeneralGraphs. To get started, follow the steps outlined below.
+
+1. In the main page of the Web Interface, go to the left sidebar 
```

Comment:
We most often write it like `web interface` (all lowercase).
"In the main page" is a bit strange as the navigation sidebar is always visible.

Reply:
Done.